src/main/java/com/develhope/java10/hellotests/HelloTestsApplication.java

```diff
@@ -7,29 +7,20 @@
 
 public class HelloTestsApplication {
 	public static void main(String[] args) {
-		File file = new File("C:\\Develhope\\HelloTests\\src\\main\\java\\com\\develhope\\java10\\hellotests\\test.txt");
+		// federico ha problemi con il pathname! Funziona solo con absoultepath
```

Comment:
in questo caso non si può usare ".\\test.txt" o qualcosa di simile?

Reply:
ci ho provato non mi trova il path... ora riprovo

Reply:
Con il path da "src\\main\\java\\com\\develhope\\java10\\hellotests\\test.txt" credo si possa risolvere